compatibility-tests/v1_31/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "v1_31"
+version = "0.1.0"
+authors = ["Jake Goulding <jake.goulding@gmail.com>"]
+
+[dependencies]
+snafu = { path = "../..", default-features = false, features = ["backtraces"] }
+
+# These versions are the last versions to support Rust 1.31
+# <none yet>
```

Comment:
Heh, looks like we spoke too soon. Evidently the backtrace crate needs to be pinned to an earlier version here.

Reply:
Yep!  Sorry, forgot to clean my compatibility-test Cargo.lock files.  I wonder if I can automate that somehow...

I found it requires 0.3.35 at most.  backtrace-sys doesn't have to be pinned.  I just pushed that fix.